lms/djangoapps/course_wiki/tests/tests.py

```diff
@@ -167,3 +167,25 @@ def test_redirect_when_not_logged_in(self):
         resp = self.client.get(course_wiki_page, follow=True)
         target_url, __ = resp.redirect_chain[-1]
         self.assertTrue(reverse('signin_user') in target_url)
+
+    @patch.dict("django.conf.settings.FEATURES", {'ALLOW_WIKI_ROOT_ACCESS': True})
+    def test_create_wiki_with_long_course_id(self):
+        """
+        Tests that the wiki is successfully created for courses that have
+        very long course ids.
+        """
+        course = CourseFactory.create(
+            org='some-very-long-org-name',
+            course='some-even-longer-course-name',
+            display_name='a-very-long-display-name-too'
+        )
```

Comment:
Can you add a comment here explicitly stating how the length of this combined string relates to https://github.com/edx/edx-platform/blob/98b20d1fc4ac1a8728572a55052279856ff50ec3/common/static/common/js/components/utils/view_utils.js#L16-L17?

If the value of `MAX_SUM_KEY_LENGTH` ever changes, it'd be nice to find this test while grokking for affected code locations.